pkg/controller/daemon/daemoncontroller_test.go

```diff
@@ -58,17 +58,22 @@ var (
 	noScheduleTaints      = []v1.Taint{{Key: "dedicated", Value: "user1", Effect: "NoSchedule"}}
 )
 
+func nowPointer() *metav1.Time {
```

Comment:
Just grep the code, there're some similar logic to get "now" pointer. I'll create a PR to clear them up by "NowP()" :).